alire_common.gpr

```diff
@@ -24,13 +24,12 @@ abstract project Alire_Common is
    case Style_Check_Mode is
       when "enabled"  => Style_Check_Switches :=
            ("-gnatwe",   -- Warnings as errors
-            "-gnatyd",   -- no DOS line terminators
-            "-gnatyI",   -- no IN mode
-            "-gnatyO",   -- all overrides
-            "-gnatyS",   -- separate lines after THEN/ELSE
-            "-gnatyu",   -- no unnecessary blank lines
-            "-gnatyx",   -- no extra parens around conditionals
-            "-gnaty-s"); -- relax fwd decl
+            "-gnatyg",   -- Standard GNAT style = -gnatyydISuxz
```

Comment:
One potential risk with gnatyg is that its definition can change in the future, which can bring some unwanted style errors.

Why not use our default style check switches for Alire crates?

Reply:
I guess I prefer to make an sporadic fix if that happens rather than being unaware of recommended style changes.